software/excalidraw.yml

```
@@ -0,0 +1,10 @@
+name: excalidraw
+website_url: https://excalidraw.com
```

See comments below about demo url.

```suggestion
website_url: https://docs.excalidraw.com/
```